code/data_structures/linked_list/xor_linked_list/xor_linked_list.cpp

```diff
@@ -25,24 +32,12 @@ class __Node
 public:
     explicit __Node(value_type value) :value_(value), around_(nullptr) {}
 
-    inline SPNode around(SPNode const &around)
-    {
-        return reinterpret_cast<SPNode>(reinterpret_cast<uintptr_t>(around) ^
-                                        reinterpret_cast<uintptr_t>(around_));
-    }
-
-    inline void around(SPNode const &prev, SPNode const &next)
-    {
-        around_ = reinterpret_cast<void *>(reinterpret_cast<uintptr_t>(prev) ^
-                                           reinterpret_cast<uintptr_t>(next));
-    }
-
     inline value_type &value()
     {
         return value_;
     }
 
-    inline value_type const &value() const
+    inline const value_type &value() const
     {
```

Comment:
While I understand that you are using STL naming conventions, I still suggest you use camel case.

Reply:
Maybe some of alias is need to change to CamelCase, but the STL naming conventions is useful for users, I found this <a href="https://stackoverflow.com/questions/44571362/what-is-the-use-of-value-type-in-stl-containers">example</a> to demonstrate what I mean for.

Reply:
:+1: